src/util/operationValidationUtils.js

```diff
@@ -37,11 +37,11 @@ function validateEvalMeasureParams(query) {
     }
   }
 
-  if (query.reportType === 'individual') {
+  if (query.reportType === 'subject') {
     const subjectReference = query.subject.split('/');
     if (subjectReference.length > 1 && subjectReference[0] !== 'Patient') {
       throw new BadRequestError(
-        `For report type 'individual', subject reference may only be a Patient resource of format "Patient/{id}".`
+        `For report type 'subject', subject reference may only be a Patient resource of format "Patient/{id}".`
```

Comment:
```suggestion
        `For report type parameter 'subject', subject reference may only be a Patient resource of format "Patient/{id}".`
```

Reply:
Just a suggestion to help avoid confusion between a report type parameter versus an actual report type.